sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -379,6 +379,17 @@ def test_factorial2_rewrite():
     assert factorial2(2*n + 1).rewrite(gamma) == \
         sqrt(2)*2**(n + S.Half)*gamma(n + Rational(3, 2))/sqrt(pi)
 
+@XFAIL
+def test_binomial_failing():
+    i, j = symbols('i, j',integer=True)
+    assert binomial(i, j).equals(binomial(i - 1, j - 1) + binomial(i - 1, j)) is False # doesn't hold for (i, j) == (0, 0)
```

Comment:
should be None and if it gives None then it isn't failing

Reply:
it gives True 😅 .
I'll change it to ` is not True`